Description: Make itk-gdcm library a static library if system gdcm is used
  In the case of using system gdcm this library is just an empty stub and by 
  making it static a lintian warning is avoided 
Author: Gert Wollny <gw.fossdev@gmail.com>


diff --git a/Modules/ThirdParty/GDCM/src/CMakeLists.txt b/Modules/ThirdParty/GDCM/src/CMakeLists.txt
index e98077c..70f4286 100644
--- a/Modules/ThirdParty/GDCM/src/CMakeLists.txt
+++ b/Modules/ThirdParty/GDCM/src/CMakeLists.txt
@@ -1,5 +1,5 @@
 if(ITK_USE_SYSTEM_GDCM)
-  add_library(ITKGDCM ITK-GDCM.cxx)
+  add_library(ITKGDCM STATIC ITK-GDCM.cxx)
   target_link_libraries(ITKGDCM gdcmDICT gdcmMSFF)
   itk_module_target(ITKGDCM)
   return()
